src/js/components/modals/ServiceFormModal.js

```diff
@@ -403,37 +405,28 @@ class ServiceFormModal extends mixin(StoreMixin) {
           return error;
         });
 
-        return (
-          <li key={path}>
-            {`${fieldId}: ${errors}`}
-          </li>
-        );
+        // Return path-prefixed error string
+        return `${fieldId}: ${errors}`;
+
       });
     }
 
     if (this.shouldForceUpdate(errorMessage)) {
       return (
-        <div className="error-field text-danger">
-          <h4 className="text-align-center text-danger flush-top">
-            App is currently locked by one or more deployments. Press the button
-            again to forcefully change and deploy the new configuration.
-          </h4>
-        </div>
+        <CollapsibleErrorMessage
+          className="error-for-modal"
+          message="App is currently locked by one or more deployments.
+            Press the button again to forcefully change and deploy the new configuration." />
```

Comment:
For these parameters that are strings and where you want a line break, it is preferred that we use this syntax, so that it doesn't cause the diff to look like this with the red background.
```
message={`App is currently...`}
```